From: Eric Anholt <eric@anholt.net>
Date: Sat, 6 Jun 2009 09:44:00 +0000 (+0000)
Subject: drm/i915: Fix build from previous commit.
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fanholt%2Fdrm-intel.git;a=commitdiff_plain;h=34c8638bb5ee130b8151c57708284bcd4972e2e1;hp=003c9ca68810bc0a63a1f3ff6beec839b197e23b

drm/i915: Fix build from previous commit.

failure to update-index after git-am --reject to hand-apply

Signed-off-by: Eric Anholt <eric@anholt.net>
---

Index: linux-2.6.29/drivers/gpu/drm/i915/i915_drv.h
===================================================================
--- linux-2.6.29.orig/drivers/gpu/drm/i915/i915_drv.h
+++ linux-2.6.29/drivers/gpu/drm/i915/i915_drv.h
@@ -142,7 +142,7 @@ typedef struct drm_i915_private {
 	unsigned int status_gfx_addr;
 	drm_local_map_t hws_map;
 	struct drm_gem_object *hws_obj;
-
+    struct resource mch_res;
 	unsigned int cpp;
 	int back_offset;
 	int front_offset;
